gammapy/estimators/points/sed.py

```diff
@@ -56,32 +56,36 @@ class FluxPointsEstimator(FluxEstimator, parallel.ParallelMixin):
     energy_edges : list of `~astropy.units.Quantity`, optional
         Edges of the flux points energy bins. The resulting bin edges won't be exactly equal to the input ones,
         but rather the closest values to the energy axis edges of the parent dataset.
-        Default is None: apply the estimator in each energy bin of the parent dataset.
-        For further explanation see :ref:`estimators`.
-    fit : `Fit`
-        Fit instance specifying the backend and fit options.
-    reoptimize : bool
-        If True the free parameters of the other models are fitted in each bin independently,
+        Default is [1, 10] TeV.
+    fit : `Fit`, optional
+        Fit instance specifying the backend and fit options. Default is None, so "scipy" is used.
+    reoptimize : bool, optional
+        If True, the free parameters of the other models are fitted in each bin independently,
         together with the norm of the source of interest
         (but the other parameters of the source of interest are kept frozen).
-        If False only the norm of the source of interest if fitted,
+        If False, only the norm of the source of interest is fitted,
         and all other parameters are frozen at their current values.
-    sum_over_energy_groups : bool
-        Whether to sum over the energy groups or fit the norm on the full energy grid.
-    n_jobs : int
-        Number of processes used in parallel for the computation. Default is one, unless
+        Default is False.
+    sum_over_energy_groups : bool, optional
+        Whether to sum over the energy groups or fit the norm on the full energy grid. Default is None.
+    n_jobs : int, optional
+        Number of processes used in parallel for the computation. Default is None, unless
         `~gammapy.utils.parallel.N_JOBS_DEFAULT` was modified. The number of jobs is
         limited to the number of physical CPUs.
-    parallel_backend : {"multiprocessing", "ray"}
-        Which backend to use for multiprocessing.
-    norm : `~gammapy.modeling.Parameter` or dict
-        Norm parameter used for the fit
+    parallel_backend : {"multiprocessing", "ray"}, optional
+        Which backend to use for multiprocessing. Default is None.
```

Comment:
Something like the description in LightCurveEstimator is maybe more clear:
Which backend to use for multiprocessing. If None, defaults to `~gammapy.utils.parallel.BACKEND_DEFAULT`.